templates/integration/integration_temp.tmpl

```diff
@@ -0,0 +1,48 @@
+package {{ .CustomData.PackageName }}
```

Comment:
What's this? In the tasks that supposed to only remove tests I see a lot of additions in git diff